modules/openshift-developer-cli-mirroring-a-supported-builder-image.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// modules/openshift-developer-cli-creating-and-pushing-a-component-from-client-os-to-the-disconnected-cluster.adoc
+
+
+= Mirroring a supported builder image
+
+. Verify that the required images tag are not imported:
++
+----
+$ oc describe is nodejs -n openshift
+----
+
+. Mirror the supported image tag to the private registry:
++
+----
+$ oc image mirror registry.access.redhat.com/rhscl/nodejs-10-rhel7:<tag> <private_registry>/rhscl/nodejs-10-rhel7:<tag>
+----
+
+. You must periodically re-import the image. Use the --scheduled flag to do it automatically:
```

Comment:
Is this step importing the image? If so, I would consider rewording a bit.

Maybe just say "Import the image" as the step.  And then have a callout on that line that says something like "You must periodically re-import the image. The `--scheduled` flag re-imports the image automatically." 

Not sure if you typically use callouts in these sections or not, but it doesn't have to be a callout.